### PR TITLE
added prompt proto

### PR DESCRIPTION
**Pull request by cdelguercio**

### CODE REVIEW

**Comment by ttulsky** [comment]
Banana rama

**Comment by cdelguercio** [comment]
> Banana rama

Bannana

**Review by ttulsky — APPROVED** [review]
Looks good

File: proto/prompt.proto [hunk 1]
```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package playground;
+
+message Prompt {
+  string title = 1;
+  string description = 2;
```
Comment (cdelguercio) [inline proto/prompt.proto:7]: what's this mean?

Comment (cdelguercio) [reply proto/prompt.proto:7]: I don't know

**Pull request closed without merging**